### PR TITLE
Install nodes and launch files

### PR DESCRIPTION
**Pull request by CameronDevine**
This pull request allows the `apriltags2_ros` nodes and launch files to be installed, so if the install space is used the files remain accessible.

### CODE REVIEW

File: apriltags2_ros/CMakeLists.txt [hunk 1]
```diff
@@ -116,3 +116,16 @@ target_link_libraries(${PROJECT_NAME}_single_image_client_node
   common
   ${catkin_LIBRARIES}
 )
+
+foreach(dir launch config)
+  install(DIRECTORY ${dir}/
```
Comment (wxmerkt) [inline apriltags2_ros/CMakeLists.txt]: Can be simplified to `install(DIRECTORY dir launch config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}`

File: apriltags2_ros/include/apriltags2_ros/common_functions.h [hunk 2]
```diff
@@ -64,7 +64,7 @@
 
 #include "apriltags2_ros/AprilTagDetection.h"
 #include "apriltags2_ros/AprilTagDetectionArray.h"
-#include "apriltag.h"
+#include "apriltags2/apriltag.h"
```
Comment (wxmerkt) [inline apriltags2_ros/include/apriltags2_ros/common_functions.h]: Where does this change come from? Is it required or optional?

Comment (CameronDevine) [reply apriltags2_ros/include/apriltags2_ros/common_functions.h]: I was not able to get the package to build without this change.

File: apriltags2_ros/CMakeLists.txt (hunk 1 above)
Comment (CameronDevine) [reply apriltags2_ros/CMakeLists.txt]: I will make this change.

File: apriltags2_ros/include/apriltags2_ros/common_functions.h (hunk 2 above)
Comment (CameronDevine) [reply apriltags2_ros/include/apriltags2_ros/common_functions.h]: I tried to build the packages again today and had to remove this for them to build. I don't know what I was doing wrong before. This has been put back to the way it was.

**Pull request closed without merging**